src/Symfony/Component/Console/Tests/Formatter/OutputFormatterTest.php

```diff
@@ -124,10 +128,9 @@ public function testDeepNestedStyles()
 
     public function testNewStyle()
     {
-        $formatter = new OutputFormatter(true);
-
         $style = new OutputFormatterStyle('blue', 'white');
-        $formatter->setStyle('test', $style);
```

Comment:
this test should be kept unchanged. If you cannot, it means there is a BC break. And that's not something we want